### PR TITLE
Jhalackova upgrade fix

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by lvicoun** [comment]
[cha-upgrade-procedure-slemicro_color_en.pdf](https://github.com/SUSE/doc-sle/files/9990272/cha-upgrade-procedure-slemicro_color_en.pdf)

File: xml/slemicro_update_procedure.xml [hunk 1]
```diff
@@ -7,82 +7,87 @@
...
    <dm:bugtracker></dm:bugtracker>
    <dm:translation>yes</dm:translation>
   </dm:docmanager>
   <abstract>
-    <para>
-    SUSE offers a command-line tool to upgrade a running system to a new version. The tool provides support for “rollback” to an older release. This chapter explains how to upgrade your system step by step.
+   <para>
+    SUSE offers a command-line tool to upgrade a running system to a new
+    version. The tool provides support for “rollback” to an older release. This
+    chapter explains how to upgrade your system step by step.
```
Comment (dariavladykina) [inline xml/slemicro_update_procedure.xml:21]:
```suggestion
    version. The tool provides support for a <quote>rollback</quote> to an older
    release. This chapter explains how to upgrade your system step by step.
```

File: xml/slemicro_update_procedure.xml [hunk 2]
```diff
@@ -7,82 +7,87 @@
...
-   </listitem>
-   <listitem>
-    <para>
-     Use of all active repositories.
-    </para>
-   </listitem>   
-  </itemizedlist>
-
-  <sect1 xml:id="sec-run-transactional-update">
+  &suse; releases new version of &slema; at regular intervals. To make it easy
```
Comment (dariavladykina) [inline xml/slemicro_update_procedure.xml:26]:
```suggestion
  &suse; releases a new version of &slema; at regular intervals. To make it easy
```

File: xml/slemicro_update_procedure.xml [hunk 3]
```diff
@@ -7,82 +7,87 @@
...
-<important>
-  <title>Upgrading to a new release and <command>wicked</command></title>
-  <para>
-    &slema; 5.3 introduced NetworkManager as the default tool to manage networking. However, when you perform an upgrade from previous releases, the default network framework remains <command>wicked</command>. Migration from <command>wicked</command> to &nm; is not supported and is not recommended. If you need to use &nm;, perform a new installation of &slema;.
-  </para>
-</important>
+  <important>
+   <title>Upgrading to a new release and <command>wicked</command></title>
+   <para>
+    &slema; 5.3 introduced Network Manager as the default tool to manage
```
Comment (dariavladykina) [inline xml/slemicro_update_procedure.xml:68]:
```suggestion
    &slema; 5.3 introduced NetworkManager as the default tool to manage
```

File: xml/slemicro_update_procedure.xml [hunk 4]
```diff
@@ -124,23 +129,75 @@
...
+    <para>
+     Run the following command:
+    </para>
+<screen>&prompt.root;transactional-update dup</screen>
+   </step>
   </procedure>
- </sect1>
 
+  <para>
+   After you complete the upgrade procedure, if needed, you can to switch from
```
Comment (dariavladykina) [inline xml/slemicro_update_procedure.xml:151]:
```suggestion
   After you complete the upgrade procedure, if needed, you can switch from
```

File: xml/slemicro_update_procedure.xml [hunk 5]
```diff
@@ -124,23 +129,75 @@
...
+<screen>&prompt.root;systemctl enable NetworkManager</screen>
+   </step>
+   <step>
+    <para>
+     If needed, configure the service according to your needs.
+    </para>
+   </step>
+   <step>
+    <para>
+     Close the <command>transactonal-update</command> shell:
```
Comment (dariavladykina) [inline xml/slemicro_update_procedure.xml:190]:
```suggestion
     Close the <command>transactional-update</command> shell:
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

File: xml/slemicro_update_procedure.xml [hunk 6]
```diff
@@ -7,82 +7,75 @@
...
  <info>
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:bugtracker></dm:bugtracker>
    <dm:translation>yes</dm:translation>
   </dm:docmanager>
   <abstract>
-    <para>
-    SUSE offers a command-line tool to upgrade a running system to a new version. The tool provides support for “rollback” to an older release. This chapter explains how to upgrade your system step by step.
+   <para>
+    SUSE offers a command-line tool to upgrade a running system to a new
```
Comment (dariavladykina) [inline xml/slemicro_update_procedure.xml:19]:
```suggestion
    &suse; offers a command-line tool to upgrade a running system to a new
```

File: xml/slemicro_update_procedure.xml [hunk 7]
```diff
@@ -7,82 +7,75 @@
...
+  </listitem>
+  <listitem>
+   <para>
+    Simple recovery if there is an error.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    It is possible to do a <quote>rollback</quote> via system tools&mdash;no
+    backup or restore needed.
```
Comment (dariavladykina) [inline xml/slemicro_update_procedure.xml:49]:
```suggestion
    Possible <quote>rollback</quote> via system tools&mdash;no backup
    or restore needed.
```

File: xml/slemicro_update_procedure.xml [hunk 8]
```diff
@@ -124,23 +117,44 @@
...
+   <figure xml:id="fig-extensions-upgrade">
+    <title>Upgrading extensions</title>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata  fileref="slemicro_upgrade_paths.svg" width="50%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="slemicro_upgrade_paths.svg" width="50%"/>
+     </imageobject>
+    </mediaobject>
```
Comment (dariavladykina) [inline xml/slemicro_update_procedure.xml:154]: Is `<textobject role="description"/>` element for alt text missing here?

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions. Thanks!

**Pull request merged**